stored_queries/ncbi_tax/ncbi_taxon_get_siblings.yaml

```diff
@@ -0,0 +1,26 @@
+# Get the array of siblings for a taxon
+# Results are limited to 10k
+name: ncbi_taxon_get_siblings
+params:
+  type: object
+  required: [key]
+  properties:
+    key:
+      type: string
+      title: Document key
+      description: Key of the taxon vertex for which you want to find siblings
+query: |
+  // First get the id of the parent node
+  let parent_id = FIRST(
+    for t in ncbi_taxon
+      filter t._key == @key
+      for p in 1..1 OUTBOUND t ncbi_child_of_taxon
+        return p._id
```

Comment:
What if @key is the root

Reply:
Added a test for key being the root, in which case the results are empty